src/data.yaml

```diff
@@ -2934,6 +2934,19 @@ jobs:
         long: 
     review: 
     rating: 
+  - name: Advanced Infrastructure
+    website: https://www.advanced-infrastructure.co.uk/
+    jobs: https://www.advanced-infrastructure.co.uk/career
+    description: Our mission is to "accelerate an informed, fair and affordable transition to a net zero energy system"
+    remote: false
+    speculative: false
```

Comment:
They have speculative applications, so this should be `true`

> Didn't find a role that suits you?
As we grow we're constantly recruiting across data, software and commercials. If you can't see a role that suits you at the moment, we'd still love to hear from you.
Send an email to [careers@advanced-infrastructure.co.uk](mailto:careers@advanced-infrastructure.co.uk)